DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: pkgdown
 Title: Make Static HTML Documentation for a Package
-Version: 2.1.0.9000
+Version: 2.1.0.9001
```

Comment:
we don't typically bump the dev version between releases.

Reply:
```suggestion
Version: 2.1.0.9000
```